backend/app.py

```
@@ -229,6 +229,37 @@ async def complete():
+    except Exception as e:
+        return handle_error(e)
+
+
+@app.route('/api/document/load/<doc_id>', methods=['GET', 'OPTIONS'])
+async def load_document(doc_id):
+    """Load document state"""
+    try:
+        if doc_id not in documents:
+            raise AIServiceError('Document not found', 404)
+        return Response(json.dumps(documents[doc_id]), mimetype='application/json')
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)